src/analysis/souffle/tests/arcs_fact_tests/integrity_simple_test.dl

```diff
@@ -0,0 +1,178 @@
+#include "taint.dl"
```

Comment:
Could we split this up into multiple tests? Or, is it too much work?

Reply:
This can be split up, and probably should be. It's gotten rather large.

Reply:
Done.